instrumentation/opentelemetry_ecto/lib/opentelemetry_ecto.ex

```diff
@@ -44,6 +99,10 @@ defmodule OpentelemetryEcto do
       sanitized version of it. This is useful for removing sensitive information from the
       query string. Unless this option is `:enabled` or a function,
       query statements will not be recorded on spans.
+    * `:db_system` - The identifier for the database management system (DBMS).
+      defaults to the mapped value of the ecto adapter used.
+      Must follow the list of well-known db systems from semantic conventions.
+      See `https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md`
```

Comment:
Would this be globally or per Adapter bases?

Reply:
Per adapter bases. I think `OpentelemetryEcto.setup/2` is meant to use just for one repo, right? And each repo can have only one Adapter (AFAIK).